pkg/package.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Is there a method to implement this that does not require maintain multiple copies of the `package.json` in the source control? That seems like a lot to do in order to add this `export`/`import` feature.

Reply:
Yes, there are a tool that convert ES Module syntax (ESM) to Common.js and include type definitions for TypeScript.

Reply:
Sorry, the question was how to not maintain multiple copies _in source control_.

For example if someone wants to contribute, how do they know which files yo modify?